spec/requests/genre_spec.rb

```diff
@@ -18,16 +19,16 @@
   end
   describe "POST /genres" do
     let!(:doorkeeper) { 
-      Doorkeeper::Application.create!(name: "Android client", redirect_uri: "", scopes: "")    
+      Doorkeeper::Application.create!(name: Faker::Lorem.sentence, redirect_uri: "", scopes: "")    
     }
     let!(:user) { 
-        User.create!(email: "test@example.com", password: "testxtest")    
+      User.create!(email: Faker::Internet.email, password: Faker::Internet.password(min_length: 8))    
     }
     let!(:token) { 
-        Doorkeeper::AccessToken.create! :application_id => doorkeeper.id, :resource_owner_id => user.id 
+        Doorkeeper::AccessToken.create!(application_id: doorkeeper.id, resource_owner_id: user.id)
     }  
     it "Create the movies" do
-      post("/genre", params: {email: "test@example.com", password: "testxtest", client_id: doorkeeper.uid, title: "test", description: "test"})
+      post("/genre", params: {email: user.email, password: user.password, client_id: doorkeeper.uid, title: Faker::Company.name , description: Faker::Lorem.sentence})
```

Comment:
Co uważasz o zrobieniu osobno `let(:params)` i wrzuceniu tam tego hasha? To tylko moja opinia, ale mogłoby ładniej wyglądać 🙂

Reply:
Spojrzę na to później :D.
Poprawiłem niektóre rzeczy o których napisałaś.